sogs/routes/__init__.py

```diff
@@ -30,6 +30,8 @@ def serve_index():
     rooms = get_readable_rooms()
     if len(rooms) == 0:
         return render_template('setup.html')
+    if not config.HTTP_SHOW_INDEX:
+        abort(http.FORBIDDEN)
```

Comment:
would `http.NOT_FOUND` be more appropriate?

Reply:
Forbidden is what web servers return when directory indexing is turned off and you try to access a directory, so seems appropriate here.